packages/tradershub/src/features/cfd/modals/MT5PasswordModal/MT5PasswordModal.tsx

```diff
@@ -0,0 +1,304 @@
+import React, { FC, useCallback, useMemo, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import {
+    useAccountStatus,
+    useActiveTradingAccount,
+    useAvailableMT5Accounts,
+    useCreateMT5Account,
+    useMT5AccountsList,
+    useSettings,
+    useTradingPlatformPasswordChange,
+} from '@deriv/api';
+import { Provider } from '@deriv/library';
+import { Button, useBreakpoint } from '@deriv/quill-design';
+import { ActionScreen, ButtonGroup, Dialog, ModalStepWrapper } from '../../../../components';
+import MT5PasswordIcon from '../../../../public/images/ic-mt5-password.svg';
+import { TMarketTypes, TPlatforms } from '../../../../types';
+import { validPassword } from '../../../../utils/password';
+import { companyNamesAndUrls, MarketTypeDetails, PlatformDetails } from '../../constants';
+import { CFDSuccess, CreatePassword, EnterPassword } from '../../screens';
+
+type TProps = {
+    marketType: TMarketTypes.SortedMT5Accounts;
+    platform: TPlatforms.All;
+};
+
+const MT5PasswordModal: FC<TProps> = ({ marketType, platform }) => {
```

Comment:
In my opinion we should use FC where there are more than 5 props.
But its upto you to use FC
```suggestion
const MT5PasswordModal = ({ marketType, platform }: {
    marketType: TMarketTypes.SortedMT5Accounts;
    platform: TPlatforms.All;
};) => {
```